readme.md

```diff
@@ -64,6 +65,7 @@ _NB: Beyond session 1, these sessions aren't in any particular order. There's so
     - [What Wal-Mart Knows About Customers' Habits](http://www.nytimes.com/2004/11/14/business/yourmoney/what-walmart-knows-about-customers-habits.html). Constance L. Hays.
     - [A Few Notes on the Culture](http://www.vavatch.co.uk/books/banks/cultnote.htm). Iain M Banks.
     - [How Far to AI-topia?](http://www.milesbrundage.com/blog-posts/how-far-to-ai-topia). Miles Brundage.
+    - [Folding Beijing](http://uncannymagazine.com/article/folding-beijing-2/. Hao Jingfang (translated by Ken Liu).
```

Comment:
Short story about a Beijing where the city "folds" every 12 hours. Each folding reveals a completely new city, where a specific socio-economic class gets to live.